index.js

```diff
@@ -643,13 +682,36 @@ function getStatusEffectNamesForVersion (supportFeature) {
   }
 }
 
+function getEntityBB (entity) {
+  const w = entity.width / 2
+  const pos = entity.position
+  return new AABB(-w, 0, -w, w, entity.height, w).offset(pos.x, pos.y, pos.z)
+}
+
+function getIntersectingEntities (bb, entityList) {
```

Comment:
`entityList.filter((e) => bb.intersects(getEntityBB(e))` would be a simpler implementation ?
what is the `e.type !== 'object'` for ?

Reply:
`e.type !== 'object'` is so we don't get pushed around by items, only entities of type `player` and `mob`
`.filter()` completely flew over my head, I'll revise it to use that instead